### ISSUE TITLE
Eves Keep Returning to Old Towns

### ISSUE THREAD

**Issue by Forman9** [jasonrohrer/OneLifeData7#1109]
This has been an ongoing theme for a while but Eves returning to old bases is in fact a VERY bad thing.

It strips the water and oil (and milk weed and wild food) for tens of thousands of tiles which basically ruins the server.

It forces players into towns they don't want to play in.

And for those who remember, this behavior also BROKE the eve spawn system for weeks one time.

There needs to be some kind of check to prevent or at least reduce the rate of Eves returning to old towns.

I think wells should honestly just eventually collapse but there are plenty of options.

**Comment by Tarr-OHOL** [comment]
People valuing towns has always been a thing due to short lineages. The issue right now is that because roads are easier to make people are able to run east to towns easier. 

I would go the other way honestly. Make pseudo-Eve an option by allowing more iron unlocks but keep food decay. Bad players will leave to make a second town and die because food decay, while more thoughtful players could make a second viable town.

**Comment by oholseven** [comment]
An Eve has a couple choices, make a new start in the wilderness, expect a good amount of early deaths as they struggle to get things going, but having a chance at a longer lineage with more resources.  Or run back to a dead town, use established resources, giving her kids an easier life, less immediate death, but likely stunting the lineage with lack of resources or the boredom that drives players in big towns to wander off for more adventure.  Maybe a simple popup message for an Eve, "Congratulations, you're an Eve!  Your mission is to establish a new civilization here.  Find a wet spring with iron mines east or west, and don't forget to choose a family name.  Good Luck Eve!"

**Comment by SoloAceMouse** [comment]
I agree with Tarr.

If this resettling is such a problem, then let pseudo-eve gameplay actually unlock iron and be able to make a viable town. This will almost invariably fail without good planning, so it naturally filters out inexperienced players.

I think resettling should be allowed in the game, especially with updates wiping fams, people should have the ability to return.

-----

>It strips the water and oil (and milk weed and wild food) for tens of thousands of tiles which basically ruins the server.

There has been exactly one town like this, to my knowledge, and it did not "ruin the server". In fact, the era of the FIRST AND ONLY town to ever survive over 10,000 tiles from the others was one of the most active periods in the game in years.

I understand the desire for primitive OHOL gameplay, but these arguments are fallacious and unfounded.

Similarly, the posts in the past on the forums about "months long towns" are absolutely untrue. The single longest lived family in OHOL wasn't even three weeks old when it died.

-----

>It forces players into towns they don't want to play in.

This is a purely subjective statement. If I want to play in a dead town, it completely invalidates this point by merit of meaning that I do want to play in it.

-----

>There needs to be some kind of check to prevent or at least reduce the rate of Eves returning to old towns.

The rate of successful resettlement in OHOL is remarkably low, with the vast majority dying out within a few generations.

Only resettles of towns which players are already invested in tend to succeed, largely due to the players wishing to live there. Resettling is already counter-balanced by trade proximity and the looting of dead towns making it difficult to restart industry there.

-----

tl;dr - Resettling is a bogey-man problem often conjured by people who either don't seem play/understand modern OHOL or used as a bad-faith argument by those who know their position is unmeritorious.

**Comment by jasonrohrer** [comment]
The problem with allowing explorers to unlock iron is that we then get back to infinite iron again.

Explorer goes out, unlocks iron, others follow, and bring all that extra iron back to town.


There are two issues fighting each other here:

1.  We need to put Eve far enough away that there are plenty of fresh resources around, and it's not practical to go back to an old town to settle.

2.  We need to put Eve close enough to everyone else that people can eventually trade with her family.


If a town died out, isn't it risky to go back and re-settle?  If it seems like if it's not risky currently, then go for it.  Eventually, once all resources around the old town are stripped, no one will be able to survive there, and this will take care of itself.

We are all the way out at 500,000 at this point, so people have been moving West over time.  It's not like they have stagnated in one spot forever..


Yesterday (Thursday), the average non-dtown, non-tutorial birth location was:

-500,755

Last Thursday, it was:

-495,148

Two Thursdays ago, it was:

-490,488


So it's moving about 5000 west every week.  Seems okay to me.
